storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/schema/Schema.java

```diff
@@ -35,4 +35,18 @@
               }
             });
   }
+
+  static Stream<RocksDbVariable<?>> streamVariables(Class<? extends Schema> schema) {
+    return Arrays.stream(schema.getDeclaredFields())
+        .filter(f -> (f.getModifiers() & Modifier.STATIC) > 0)
```

Comment:
```suggestion
        .filter(f -> Modifiers.isState(f.getModifiers()))
```
and could apply the same change above in `streamColumns` because apparently you should use the `java.lang.reflect.Modifiers` class to decode modifiers...